src/icu4c.mk

```diff
@@ -13,7 +13,7 @@ $(PKG)_URL      := https://ssl.icu-project.org/files/$(PKG)/$($(PKG)_VERSION)/$(
 $(PKG)_DEPS     := cc
 
 define $(PKG)_UPDATE
-    $(WGET) -q -O- 'http://bugs.icu-project.org/trac/browser/icu/tags' | \
+    $(WGET) -q -O- 'https://ssl.icu-project.org/trac/browser/icu/tags' | \
```

Comment:
This site doesn't work for me with https, I get connection refused.

Reply:
Indeed, thanks! — Their SSL support is not operating, even though it has been for many years.
I've undone this update.